tests/unit/drivers/complex_vlan.py

```diff
@@ -43,6 +43,9 @@ def config_initialize():
                         '{"name":"1", "switch":"test"}, ' \
                         '{"name":"2", "switch":"test"}]')
             cfg.set('driver complex_vlan', 'trunk_ports', '[]')
+            cfg.add_section('database')
+            cfg.set('database', 'sqlite', 'sqlite:///:memory:')
+            cfg.set('database', 'postgres', 'postgresql://postgres:postgres@localhost/haas')
```

Comment:
Please do not hard code the postgres uri; until we can properly parametrize this (which we'll want at some point) I'd like to keep sqlite as the test database, so people can easily run the test suite on their laptops.